tools/code_format/envoy_build_fixer.py

```diff
@@ -2,7 +2,7 @@
 
 # Enforces:
 # - License headers on Envoy BUILD files
-# - envoy_package() or envoy_extension_package() top-level invocation for standard Envoy package setup.
+# -  or envoy_extension_package() top-level invocation for standard Envoy package setup.
```

Comment:
this comment is a bit off now, not sure envoy_package() should've been removed from here?

Reply:
This is actually a file in the Envoy Core. I added a comment about the `envoy_mobile_package`.